migration/20210622-remove-rbd.sql

```diff
@@ -0,0 +1,44 @@
+BEGIN;
```

Comment:
This seems like it should work, but the convention for CM and core thus far has been to wrap transactions inside a DO block as a dollar-quoted literal, like:
```
DO $$
  BEGIN
    ...
  END;
$$;

Reply:
Interesting. I didn't notice that was the convention before. 

I've never seen a SQL `DO` statement before TBH. It must be something PG specific.

Reply:
Neat, the postgres `DO` block is an SQL extension that lets you execute a function in a procedural language. [Docs are here](https://www.postgresql.org/docs/9.6/sql-do.html). Good to know for the future that you can actually write a script to make some updates. Thanks for pointing that out! 

Looked through the code that runs the updates and it looks like it automatically wraps everything in a transaction anyway, so the transaction I had as part of this script was redundant. I removed it so the update is following the convention the CM was using.